sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentField.java

```diff
@@ -15,72 +15,12 @@
  * An object representing the content and location of a field value.
  */
 @Immutable
-public final class DocumentField {
+public final class DocumentField extends TypedDocumentField<Object> {
     /*
      * Data type of the field value.
      */
     private DocumentFieldType type;
```

Comment:
Should this property also be in the superclass? The type information should be available for `TypedDocumentField` too.